### PR TITLE
Fix #371, #373, #374: Update exploration json to latest versions

### LINKED ISSUE #371: Fix image rendering for interactions
The third state in the fractions lesson demonstrated not having proper image rendering support (the bullet items were empty).

### LINKED ISSUE #373: Fix missing images in explorations
Fractions explorations do not render images at all.

### LINKED ISSUE #374: Audio voiceovers are not playing
Audio voiceovers are not working in fractions lessons.

### PR DESCRIPTION
**Pull request by BenHenning**
Fix #371.
Fix #373.
Fix #374.

All three of these issues were caused by the audio & image references being wrong in the fractions lessons, likely due to them being old. This PR updates fractions lessons so that each work correctly. It also adds some nice-to-have missing fields to all lessons, such as version and ID.

~This also updates StateRetriever to remove legacy parsing code now that the fractions explorations are using the latest schema version.~
StateRetriever can't be updated yet since some of the questions structures appear to rely on this legacy parsing.

Verified all domain tests are passing.

### CODE REVIEW

File: domain/src/main/assets/ratios_exploration0.json [hunk 1]
```diff
@@ -4703,10 +4707,10 @@
...
+  "init_state_name": "Introduction",
   "param_specs": {},
-  "title": "What is a Ratio?",
   "objective": "Learn about how ratios can be used to show the relative relationship between quantities.",
-  "language_code": "en",
-  "init_state_name": "Introduction",
+  "title": "What is a Ratio?",
   "correctness_feedback_enabled": true
-}
+}
```
Comment (seanlip) [inline domain/src/main/assets/ratios_exploration0.json]: Nit: Is the deletion of the newline at EOF intentional? Ditto in other files.

**Review by seanlip — APPROVED** [review]
LGTM, thanks! (One minor nit that recurs through several files, but feel free to merge after that.)

File: domain/src/main/assets/ratios_exploration0.json (hunk 1 above)
Comment (BenHenning) [reply domain/src/main/assets/ratios_exploration0.json]: Nope, was missed. Fixed--thanks.

**Comment by BenHenning** [comment]
Thanks!

**Pull request merged**